index.html

```diff
@@ -16448,17 +16461,21 @@ <h4>`summary` Element Accessible Name Computation</h4>
       </section>
       <section>
         <h4>`figure` Element Accessible Name Computation</h4>
+        <p>
+          A <a href="#el-figcaption">`figcaption`</a> provides additional details related to its parent 
+          <a href="#el-figure">`figure`</a> element. A `figcaption` does not take part in the 
+          <a data-cite="accname-1.2/#dfn-accessible-name">accessible name</a> or <a data-cite="accname-1.2/#dfn-accessible-description">accessible description</a>
+          computation, unless explicitly referenced by an author. In such an instance, user agents SHOULD NOT expose the details relationship between the
+          `figure` `figcaption` elements, and instead expose the `figcaption` as the element's accessible name or description, as specified by the author.
```

Comment:
"In such an instance" is unclear to me. I think this might be the intention:

```suggestion
          computation, unless explicitly referenced by an author. When an author uses `aria-labelledby` or `aria-describedby` to reference the first `figcaption` element from a`figure` element, user agents SHOULD NOT expose the details relationship between the
          `figure` and `figcaption` elements, and instead expose the `figcaption` as the element's accessible name or description, according to the <a href="https://w3c.github.io/accname/#mapping_additional_nd">AccName Name and Description Computation.</a>
```
Sorry I think you'll need to change that link above.

Reply:
i don't think we need to link to accName here, since the following naming steps do that already.  but i'm pulling in the other suggetions